test/testcafe/spec/SignInWebAuthnView_spec.js

```diff
@@ -13,6 +13,7 @@ const mock = RequestMock()
   .respond(launchWebAuthnOption);
 
 fixture('Sign in with Okta Verify is required')
+  .meta('gen3', false)
```

Comment:
@lesterchoi-okta this fixture has the same name as `SignInDeviceView_spec.js` so was initially missed since I was using the fixture name as the key in the Set. (I've updated the list in the ticket)